ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -136,6 +136,37 @@ const primalOutputStrings = {
   },
 };
 
+const statueBaitResponse = (data, _, output) => {
+  // cactbot-builtin-response
+  output.responseOutputStrings = {
+    inner: {
+      en: 'Bait Inner Statue',
+    },
+    outer: {
+      en: 'Bait Outer Statue',
+    },
+    unknown: {
+      en: '???',
+    },
+    sides: Outputs.sides,
+  };
+  data.statueLaserCount = data.statueLaserCount + 1 || 1;
+  let response;
+  if (data.statueLaserCount === data.statueTetherNumber)
+    response = { alertText: output[data.statueDir]() };
+  else
+    response = { infoText: output.sides() };
```

Comment:
I think this is the part that I'm least sure about in this patch.  Am I right in reading it will just call `Sides, Sides, Sides, Bait Inner Statue!`

I think it might be nicer to count them out and say `1, 2, 3, Bait Inner Statue`

Reply:
Does that make sense when the order ends up being, for example, `1, Bait Inner Statue, 3, 4`?

Additionally, in the same sense, should the knockback direction influence the laser bait direction?

Reply:
Re: order.  It could say "2 (Bait Inner)" with alert/alarm vs info if you think it's asymmetrical to not have the number there.  I could also see having this same trigger call out the number with info and the "Bait Inner Statue" with alert.  For tts users, that would say "bait inner statue", but visual text users would see both.  Do any of those options sound better to you?

Re: knockback direction.  I think this is one of those strat-specific things.  I think it'd be easier to just say "bait" here and trust the player to go to the right spot.  My group does haircut-specific directions usually.